<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiroDeployDialog</class>
 <widget class="QDialog" name="MiroDeployDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deploy MIRO Model</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modes to deploy</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="baseBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Base</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="hypercubeBox">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Hypercube</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Execution environment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="targetEnvBox">
       <item>
        <property name="text">
         <string>Multi user</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Local multi user (uses inputDir)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Single user</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="testDeployButton">
         <property name="text">
          <string>Test Deploy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deployButton">
         <property name="text">
          <string>Deploy</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
